src/main/java/org/jitsi/videobridge/simulcast/SimulcastStream.java

```diff
@@ -243,4 +243,29 @@ public void askForKeyframe()
     {
         simulcastReceiver.askForKeyframe(this);
     }
+
+    @Override
```

Comment:
These need javadocs, and formatting fixes. I don't think we have one-line ifs anywhere else.